src/EA.Weee.Web/Areas/Aatf/Views/Shared/EditorTemplates/SubmittedDatesFilterViewModel.cshtml

```diff
@@ -3,47 +3,51 @@
 @model EA.Weee.Web.Areas.Aatf.ViewModels.ManageEvidenceNoteViewModel
 
 <div class="govuk-form-group govuk-!-width-one-half govuk-!-margin-left-2 govuk-!-padding-top-0 govuk-!-padding-bottom-1">
-    @Html.Label("Filter by", new { @class = "govuk-label--s" })
+    <span class = "govuk-label--s" aria-labelledby="filter-to-label">
+        Filter by
+    </span>
 </div>
 
 <div class="govuk-form-group govuk-!-width-one-full govuk-!-padding-left-0 govuk-!-padding-top-0">
     
     <div class="govuk-grid-row ">
         
         <div class="govuk-grid-column-two-thirds">
-            <div class="govuk-grid-column-one-third govuk-!-padding-left-2 govuk-!-padding-right-0">                
+            <div class="govuk-grid-column-one-third govuk-!-padding-left-2 govuk-!-margin-right-0">                
                 
                 <div class="govuk-form-group govuk-!-padding-top-20 @Html.Gds().FormGroupClass(m => m.SubmittedDatesFilterViewModel.StartDate)">
                     @Html.Gds().LabelFor(m => m.SubmittedDatesFilterViewModel.StartDate, new { @class = "govuk-label--s" }, false)
                     @Html.Gds().ValidationMessageFor(m => m.SubmittedDatesFilterViewModel.StartDate)
 
                     @if (Model.SubmittedDatesFilterViewModel.StartDate.Equals(default(DateTime)))
                     {
-                        @Html.Gds().TextBoxFor(m => m.SubmittedDatesFilterViewModel.StartDate, new { @maxlength = CommonMaxFieldLengths.DefaultString, @type = "Date" }, false)
+                        @Html.Gds().TextBoxFor(m => m.SubmittedDatesFilterViewModel.StartDate, new { @maxlength = CommonMaxFieldLengths.DefaultString, @type = "Date", @style = "width:175px;" }, false)
                     }
                     else
                     {
-                        @Html.Gds().TextBoxFor(m => m.SubmittedDatesFilterViewModel.StartDate, new { @maxlength = CommonMaxFieldLengths.DefaultString, @type = "Date" }, false, "{0:yyyy-MM-dd}")
+                        @Html.Gds().TextBoxFor(m => m.SubmittedDatesFilterViewModel.StartDate, new { @maxlength = CommonMaxFieldLengths.DefaultString, @type = "Date", @style = "width:175px;"}, false, "{0:yyyy-MM-dd}")
```

Comment:
As the same width is used a few times, id probably create a css style .weee-evidence-filter-width or something similar

Reply:
okay